demos/asm-mcp-multi-clusters/README.md

```diff
@@ -0,0 +1,17 @@
+# Resources used during IstioCon 2022 for the Anthos Service Mesh (ASM) workshop
+
+The pre-provisioned infrastructure of this workshop is in the [terraform folder](./terraform).
+
+The actual instructions of the workshop could be found [here](./instructions/en.md).
+
+Here is the architecture diagram for this setup:
+![Architecture diagram](./instructions/img/architecture.png)
+
```

Comment:
could we also mention and have a link for BoA

Reply:
Good idea, done.